extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -70,16 +69,18 @@ export class SchemaCompareMainWindow {
 	private targetName: string;
 	private scmpSourceExcludes: mssql.SchemaCompareObjectId[];
 	private scmpTargetExcludes: mssql.SchemaCompareObjectId[];
+	private schemaCompareService: mssql.ISchemaCompareService;
 
 	public sourceEndpointInfo: mssql.SchemaCompareEndpointInfo;
 	public targetEndpointInfo: mssql.SchemaCompareEndpointInfo;
 
-	constructor() {
```

Comment:
can put the 'private' keyword in the constructor to avoid the extra definition above